exatomic/core/universe.py

```diff
@@ -349,3 +349,59 @@ def basis_function_contributions(universe, mo, mocoefs='coef',
         return joined
     else:
         raise NotImplementedError("not clever enough for that.")
+
+
+def to_cube(universe,file_name='output',field_number=0)
+    """
+    Provided a universe with atom and field attributes, write to a file 
+    in cube format for a single 3D scalar field.
+    
+    Warning:
+        This function is not fully tested yet!
+
+    Args:
+        universe (:class:`~exatomic.core.universe.Universe`): a universe
+        file_name (str): name of the output file without file extension
+        field_number (int): number of the single field starting with 0
+
+    Returns:
+        None
+    """
+    uni = universe
```

Comment:
It might be simpler to rename the argument on line 354 from `universe` to `uni`. Or is there another reason for this that I missed?